pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/Pql2Compiler.java

```diff
@@ -52,7 +52,6 @@
  * PQL 2 compiler.
  */
 public class Pql2Compiler implements AbstractCompiler {
```

Comment:
Please add ThreadSafe annotation so that next time we don't add such changeable member variable.

Reply:
I am not sure if it is thread safe.

Reply:
No member variable, ErrorListener is thread safe, so I think it is thread safe. Also, we already assume it is thread safe in the callers.